src/app/meditation/meditation.component.ts

```diff
@@ -397,6 +434,13 @@ export class MeditationComponent {
     return Math.round(val);
   }
 
+  /**
+   * Substitute for default '%' operator. '%' is not working properly in JS.
```

Comment:
That's not true. `%` is the [Remainder operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder_()) which is just different than the Modulo operator.